### DISCUSSION TITLE
Infineon Tricore support for Zephyr

### DISCUSSION THREAD

**Discussion by Franky45** [zephyrproject-rtos/zephyr#81374] [Q&A]
Hi, does Zephyr supports Infineon Aurix series 397x Tricore processor architecture
[https://www.infineon.com/cms/en/product/microcontroller/32-bit-tricore-microcontroller/32-bit-tricore-aurix-tc3xx/](url)

**Comment by JarmouniA** [comment]
> Hi, does Zephyr supports Infineon Aurix series 397x Tricore processor architecture [https://www.infineon.com/cms/en/product/microcontroller/32-bit-tricore-microcontroller/32-bit-tricore-aurix-tc3xx/](url)

Supported Infineon SoCs https://github.com/zephyrproject-rtos/zephyr/tree/main/soc/infineon

**Reply by Franky45** [reply]
so, as of now zephyr was not supporting the Tricore processor Architecture

**Reply by JarmouniA** [reply]
> so, as of now zephyr was not supporting the Tricore processor Architecture

Supported architectures https://github.com/zephyrproject-rtos/zephyr/tree/main/arch
